lib/fluent/plugin/filter_record_modifier.rb

```diff
@@ -82,7 +82,12 @@ def filter(tag, time, record)
       tag_parts = @has_tag_parts ? tag.split('.') : nil
 
       @map.each_pair { |k, v|
-        record[k] = v.expand(tag, time, record, tag_parts)
+        begin
+          record[k] = v.expand(tag, time, record, tag_parts)
+        rescue
+          $log.error("filter_record_modifier::filter - exception in expression: '#{v.getcode}'")
```

Comment:
https://github.com/repeatedly/fluent-plugin-record-modifier/blob/4512c5ba3aa846dcae5cc343b124f0ac8f15a6a7/lib/fluent/plugin/filter_record_modifier.rb#L76

config's `record` elements has original code so we don't need to store code in `DynamicExpander`.